docs/docs/guides/wallet/transactions.md

```diff
@@ -7,6 +7,193 @@ sidebar_label: 'Tutorial: Sending Transactions'
 
 This guide provides insights into sending transactions using web3.js, covering various scenarios from utilizing a local wallet to sending raw transactions.
 
+:::note
+Web3.js uses transactions type 2 by default
+:::
+
+## Transaction Type 0 (Legacy)
+
+### Raw Transaction 
+
+A Legacy Transaction refers to a transaction that was created using an older version of Ethereum's transaction format, also known as "transaction type 0". This transaction format was used before the EIP-1559 upgrade, which was implemented in August 2021.
+
+```ts
+import { Web3 } from"web3";
+
+const web3 = new Web3("https://rpc2.sepolia.org");
+
+async function txLegacy() {
+  const wallet = web3.eth.wallet.add("YOUR_PRIVATE_KEY"); //make sure you have funds
+
+  const sender = wallet[0].address;
+  const recipient = "0x807BFe4940016B5a7FdA19482042917B02e68359";
+  const value = 1; //wei
+  const nonce = await web3.eth.getTransactionCount(sender);
+  const gas = 21000;
+  const gasPrice = await web3.eth.getGasPrice();
+
+  const tx = {
+    from: sender,
+    to: recipient,
+    value,
+    nonce,
+    gas,
+    gasPrice,
+    // highlight-next-line
+    type: 0,
+  };
+
+  const txReceipt = await web3.eth.sendTransaction(tx);
+  console.log("Tx hash", txReceipt.transactionHash);
+}
+
+txLegacy();
+```
+
+### ERC20 Interaction
+
+```ts
+import { Web3 } from "web3";
+
+const web3 = new Web3("https://rpc2.sepolia.org");
+
+async function transfer() {
+  //initialize wallet
+  const wallet = web3.eth.wallet.add("PRIVATE_KEY");
+
+  //initialize contract
+  const myERC20 = new web3.eth.Contract(ABI, ADDRESS);
+
+  //send transfer and specify the type
+  const txReceipt = await myERC20.methods.transfer(TO, VALUE).send({
+    from: wallet[0].address,
+    // highlight-next-line
+    type: 0,
+  });
+
+  console.log(txReceipt.transactionHash);
+}
+
+transfer();
+```
+
+## Transaction Type 1 (EIP-2930)
+
+This EIP was introduced in April 2021, it introduces a feature called 'Transaction Type and Access List.' This improvement allows saving gas on cross-contract calls by declaring in advance which contract and storage slots will be accessed.
```

Comment:
```suggestion
This EIP was introduced in April 2021, it introduces a feature called 'Access List.' This improvement allows saving gas on cross-contract calls by declaring in advance which contract and storage slots will be accessed.
```

transaction type envelop was introduced in https://eips.ethereum.org/EIPS/eip-2718